source/opt/instruction.h

```diff
@@ -83,29 +84,30 @@ class Instruction : public utils::IntrusiveNodeBase<Instruction> {
   using iterator = std::vector<Operand>::iterator;
   using const_iterator = std::vector<Operand>::const_iterator;
 
-  // Creates a default OpNop instruction.
+  // This exists solely for containers that can't do without. Should be removed.
```

Comment:
Should still keep the comment about what it does, even if we want to eventually remove it.